File: chapter1/challenge1/java-problems/Sums.java
import java.util.Scanner;

public class Sums {
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);

		int tc, a, b; 
		tc = sc.nextInt(); // number of test cases 
	 	while (tc-- > 0) { // shortcut to repeat until 0
			a = sc.nextInt();
			b = sc.nextInt(); // compute answer 
			System.out.println(a + b); // on the fly  
		}

	}
	
}

File: chapter1/challenge1/c-problems/sums.c
#include <stdio.h>

int main() {

int tc, a, b;
scanf("%d", &tc); // number of test cases
while (tc--) { // shortcut to repeat until 0 
	scanf("%d %d", &a, &b); // compute answer 
	printf("%d\n", a + b); // on the fly 
	}
	return 0;
}

